### ISSUE TITLE
Cante100 partial download of melody and notes wrong file structure

### ISSUE THREAD

**Issue by magdalenafuentes** [mir-dataset-loaders/mirdata#357]
When partially downloading `cante100` files they end up under the `cante100` folder instead of `cante100_automaticTranscription`:

```
>>> dataset.download(partial_download=["melody", "notes"])
Starting to download ['melody', 'notes'] to folder /Users/mf3734/mir_datasets/cante100
> downloading melody
32.0MB [00:50, 666kB/s]                                                                                                                                                              
> downloading notes
368kB [00:03, 112kB/s]                                                                                                                                                               

        This loader is designed to load the spectrum, as it is available for download.
        However, the loader supports audio as well. Unfortunately the audio files of the 
        cante100 dataset are not available for free download, but upon request. However, 
        you can request de audio in both links here: 
        ==> http://www.cofla-project.com/?page_id=208
        ==> https://zenodo.org/record/1324183
        Then, locate the downloaded the cante100audio folder like this:
            > cante100/
                > cante100_spectrum/
                ... (rest of the annotation folders)
                > cante100audio/
        Remember to locate the cante100 folder to /Users/mf3734/mir_datasets/cante100

>>> track.notes
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/mf3734/Documents/repos/mirdata/mirdata/utils.py", line 215, in __get__
    value = obj.__dict__[self.func.__name__] = self.func(obj)
  File "/Users/mf3734/Documents/repos/mirdata/mirdata/datasets/cante100.py", line 279, in notes
    return load_notes(self.notes_path)
  File "/Users/mf3734/Documents/repos/mirdata/mirdata/datasets/cante100.py", line 362, in load_notes
    raise IOError("notes_path {} does not exist".format(notes_path))
OSError: notes_path /Users/mf3734/mir_datasets/cante100/cante100_automaticTranscription/051_BernardaDeUtrera_Buerias.notes.csv does not exist
```

I don't know if that's the case for other loaders.

**Comment by genisplaja** [comment]
Found it! The issue was indeed with the `REMOTES` dictionary. The partial download worked perfectly when I arranged the destination directories like that: 
```
destination_dir='cante100_spectrum',  # relative path for where to unzip the data, or None
```
I tested it for the whole download and it worked well. I downloaded different partial combinations and everything works fine! I will create a PR with the fix. I will check if there are more datasets that fail for that, but in this case it was my fault for not specifying the destination directory when writing the cante100 loader...

**Comment by magdalenafuentes** [comment]
That's great Genís! Thanks so much!

Nobody’s fault, I think we just weren't checking for this. If you can, as part of your PR include a comment in the loader's template in `CONTRIBUTE.md` explaining that partial downloads might need specific destination directories.

**Comment by magdalenafuentes** [comment]
Closed via #361
